packages/workspace-store/src/schemas/v3.1/strict/example.ts

```diff
@@ -1,20 +1,26 @@
 import { Type } from '@scalar/typebox'
 
+import { compose } from '@/schemas/compose'
+import { XIsDisabled } from '@/schemas/extensions/example/x-is-disabled'
+
 /**
  * An object grouping an internal or external example value with basic summary and description metadata. This object is typically used in fields named examples (plural), and is a referenceable alternative to older example (singular) fields that do not support referencing or metadata.
  *
  * Examples allow demonstration of the usage of properties, parameters and objects within OpenAPI.
  */
-export const ExampleObjectSchemaDefinition = Type.Object({
-  /** Short description for the example. */
-  summary: Type.Optional(Type.String()),
-  /** Long description for the example. CommonMark syntax MAY be used for rich text representation. */
-  description: Type.Optional(Type.String()),
-  /** Embedded literal example. The value field and externalValue field are mutually exclusive. To represent examples of media types that cannot naturally represented in JSON or YAML, use a string value to contain the example, escaping where necessary. */
-  value: Type.Optional(Type.Any()),
-  /** A URI that identifies the literal example. This provides the capability to reference examples that cannot easily be included in JSON or YAML documents. The value field and externalValue field are mutually exclusive. See the rules for resolving Relative References. */
-  externalValue: Type.Optional(Type.String()),
-})
+export const ExampleObjectSchemaDefinition = compose(
+  Type.Object({
+    /** Short description for the example. */
+    summary: Type.Optional(Type.String()),
+    /** Long description for the example. CommonMark syntax MAY be used for rich text representation. */
+    description: Type.Optional(Type.String()),
+    /** Embedded literal example. The value field and externalValue field are mutually exclusive. To represent examples of media types that cannot naturally represented in JSON or YAML, use a string value to contain the example, escaping where necessary. */
+    value: Type.Optional(Type.Any()),
+    /** A URI that identifies the literal example. This provides the capability to reference examples that cannot easily be included in JSON or YAML documents. The value field and externalValue field are mutually exclusive. See the rules for resolving Relative References. */
+    externalValue: Type.Optional(Type.String()),
+  }),
+  XIsDisabled,
+)
```

Comment:
In the api client store we added our custom specification extensions all over the place and it ended up being super confusing.

Also, eventually we might want to use those to validate any file, and an OpenAPI document is still valid when someone uses a specification with another structure.

So what I’m thinking is: Should we put the OpenAPI schemas in one place and do the `compose` in another file, so we have both (the original OpenAPI schema and the OpenAPI schema + our extensions).

Wdyt?

Reply:
Or we can just add the wild card to our types. Like so:
<img width="1068" height="689" alt="image" src="https://github.com/user-attachments/assets/6fb16a40-4bf7-4c0f-b4e5-cff11425c8ee" />
best of all worlds?